src/docker-osx-dev

```diff
@@ -956,12 +958,19 @@ function configure_includes {
 # Runs the docker-osx-dev script to kick off file watching and syncing.
 #
 function watch_and_sync {
+  sync_only
+  watch
+}
+
+#
+# Runs the docker-osx-dev script to syncing one time.
+#
+function sync_only {
```

Comment:
Given that the other function is called `watch`, perhaps `sync` would be a more analogous name?